src/translations/en.json

```diff
@@ -122,6 +122,9 @@
     "CSVFileCommentContingencyList2": "# Line 1,ID_line1",
     "CSVFileCommentContingencyList3": "# Transformer 2,ID_transformer2",
     "CSVFileCommentContingencyList4": "# N-2 composite,ID_line1|ID_generator1",
+    "contingencyListCreationError": "An error occurred while creating the contingency list: {name}",
+    "contingencyListEditingError": "An error occurred while editing the contingency list: {name}",
+    "contingencyListCreation": "contingencyListCreation",
```

Comment:
?

Reply:
that's the csv file name translation

Reply:
Why do you need a translation for a the file name ?